Fix .gitignore

AmirHosseinKarimi opened the pull request:
Add required files to git

xtrime-ru commented:
No, you cant use madelineProto.phar. Because both libs uses composer it will be two composer autoloaders.

AmirHosseinKarimi commented:
No, I didn't remove `composer.phar`. VSCode added a new empty line to the end of the file.
I just removed `.*` and added these lines:
```
.mysql/
.env
.env.docker
```

xtrime-ru commented:
What is purpose of this changes?

AmirHosseinKarimi commented:
In the previous version of `.gitignore` all files which started via `.`(dot) has been ignored. But there are some files started via `.` which need to be kept in git changes. for example: `.env.example` & `.env.docker.example` and also other dot files which may add in feature updates or by users.
We should ignore specified files or directories only, which should not keep in git changes. Not all files!

xtrime-ru commented:
.env.example,  .env.docker.example .gitignore and other nessesary files added to git. There is no issues to add file in git manually, even its ignored by .gitignore rules. Also files and foders can be unignored in .gitignore with "!" sign...

AmirHosseinKarimi commented:
You right. But the concept is to ignore specific files, not to ignore all files and if you want to include any file, so exclude it in .gitignore!
Anyway, It's not a huge change to spend more time on it.
